iphone/cli/hooks/frameworks.js

```diff
@@ -166,6 +166,7 @@ class FrameworkManager {
 	 *
 	 * @param {Object} hookData - Data from the Xcode project hook
 	 * @param {Function} callback - Callback function
+	 * @return {undefined}
```

Comment:
Undefined? Is this the appropriate way to document guarded return statements?

Reply:
It is actually undefined because the callback should have no return value and if a function has no specified return value it's undefined. Same goes for the "else" branch. There is no explicit return value so it will be undefined

Reply:
FWIW, I never define a `@return` when the function does not explicitly have a `return` statement.

Reply:
Yeah usually I do that too. In this case the function has a return statement `return callback()` inside an if-branch to return early at the top of the function to leave out the else branch and our linting rules complained about a missing return JSDoc.